grunt-sections/flow.js

```
@@ -3,6 +3,11 @@
 var fs = require('fs');
 var semver = require('semver');
 var shell = require('shelljs');
+let wnpm;
+
+try {
+  wnpm = require('wnpm-ci');
+} catch (e) {}
```

You should ignore exception only if e.code === 'MODULE_NOT_FOUND', otherwise rethrow.